### ISSUE TITLE
Windows instructions on Discourse are out of date

### ISSUE THREAD

**Issue by SayusiAndo** [CanonicalLtd/multipass#963]
I followed the description about how to use Virtualbox and I got the following error message:

Internet says that I need to import Hyper-V package, which, AFAIK, is not possible if Hyper-V is disabled, which is pre-requisite of using Virtualbox.

Any workaround for this, or what is that I'm doing wrong here?

```
PS C:\Users\Andras Csanyi> multipass start sdf -vvv
Starting sdf /[2019-08-13T22:34:59.559] [debug] [sdf] cmdlet: 'Get-VM -Name sdf | Select-Object -ExpandProperty State'

 /[2019-08-13T22:34:59.606] [debug] [sdf] Get-VM : The term 'Get-VM' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ Get-VM -Name sdf | Select-Object -ExpandProperty State
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (Get-VM:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

 /[2019-08-13T22:34:59.606] [debug] [sdf] cmdlet exit status is 'false'

 /[2019-08-13T22:34:59.606] [debug] [sdf] cmdlet: 'Get-VM -Name sdf | Select-Object -ExpandProperty State'

 /[2019-08-13T22:34:59.653] [debug] [sdf] Get-VM : The term 'Get-VM' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ Get-VM -Name sdf | Select-Object -ExpandProperty State
+ ~~~~~~
    + CategoryInfo          : ObjectNotFound: (Get-VM:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

 /[2019-08-13T22:34:59.653] [debug] [sdf] cmdlet exit status is 'false'

 /[2019-08-13T22:34:59.653] [debug] [sdf] cmdlet: 'Get-VM -Name sdf | Select-Object -ExpandProperty State'

 /[2019-08-13T22:34:59.715] [debug] [sdf] Get-VM : The term 'Get-VM' is not recognized as the name of a cmdlet, function, script file, or operable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ Get-VM -Name sdf | Select-Object -ExpandProperty State
+ ~~~~~~

```

**Comment by Saviq** [comment]
Hi @SayusiAndo :)

On the contrary, VirtualBox _conflicts_ with Hyper-V. The errors above indeed show you have Hyper-V disabled, so that is fine. But did you switch Multipass to the VirtualBox backend?

See [our latest release notes](https://discourse.ubuntu.com/t/announcing-the-multipass-0-8-0-beta-release/11976) for more info, but basically you need to issue `multipass set local.driver=virtualbox`, from an Administrator console.

We're currently working (#948) on much better user experience when it comes to selecting the default backend, as well us trying to help the user with any problems.

**Comment by SayusiAndo** [comment]
Thanks for your answer @Saviq! Then I made a mistake, because I did not execute the command you refer to. :) 

I followed instructions can be found [here](https://discourse.ubuntu.com/t/installing-multipass-for-windows/9547/5), but what you mention is not part of the description, moreover the release notes suggest that registry hack is not needed. Am I right?

**Comment by townsend2010** [comment]
@SayusiAndo,

Yes, those instructions are a bit dated now.  Not your fault as we failed to update those.  I will work on updating that page to the reflect the new way to set the driver as @Saviq.

Correct, no registry hack is needed nor should be done.  Just use that `multipass` command line referred to above.

I will modify this bug to say our Windows instructions on Discourse are out of date and will close it once I updated that today.

Thanks again for your feedback on Windows!  It is quite helpful :smile:
